addons/crm_iap_lead_website/models/crm_reveal_rule.py

```diff
@@ -113,13 +113,13 @@ def _compute_lead_count(self):
     def action_get_lead_tree_view(self):
         action = self.env["ir.actions.actions"]._for_xml_id("crm.crm_lead_all_leads")
         action['domain'] = [('id', 'in', self.lead_ids.ids), ('type', '=', 'lead')]
-        action['context'] = dict(self._context, create=False)
+        action['context'] = dict(self.env.context, create=False, group_by=[])
```

Comment:
So the right fix in this case would be
```suggestion
        action['context'] = dict(action.get('context', {}), create=False)
```
That way, the client will update its general context (not the action's specific context) with the action's context and an extra modification.

Reply:
Ok i'll modify this, but just to know what's the difference between general context and action's context ? (i'm new here at odoo)